### PR TITLE
Publish version 0.1.1.2 for Microsoft.Teammate

### PR DESCRIPTION
**Pull request by supermem613**
- [ Yes ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ Yes ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ Yes ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ Yes ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ Yes ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/49584)

### CODE REVIEW

File: manifests/m/Microsoft/TeamMate/0.1.2.0/Microsoft.TeamMate.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Microsoft.TeamMate
+PackageVersion: 0.1.2.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+ReleaseDate: 2022-02-28
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/microsoft/TeamMate/releases/download/0.1.2-ci.4%2BBranch.main.Sha.d7c0ecf5d886326ecd14eab6c9b6283000aaef1d/Microsoft.Tools.TeamMate.Production.msi
+  InstallerSha256: 6c416ba19c8ef5f1174e678aae599d5bc1e9668b327e2002e548aecf32767941
+  ProductCode: '{E1DACC85-6D0E-462B-9EB2-C54EA82B82DC}'
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/TeamMate/0.1.2.0/Microsoft.TeamMate.installer.yaml:13]:
```suggestion
  ProductCode: '{43E9AFD4-0EF5-4E19-B016-1D0758575B44}'
```

Comment (ItzLevvie) [reply manifests/m/Microsoft/TeamMate/0.1.2.0/Microsoft.TeamMate.installer.yaml:13]: Use https://github.com/microsoft/winget-create as part of your CI/CD.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by Corlobin — APPROVED** [review]

**Comment by supermem613** [comment]
Anybody? Is there any write-access contributor that can look at this?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> Anybody? Is there any write-access contributor that can look at this?

Please close and re-open the PR to re-run the pipelines.

**Pull request merged**